api/oc_main.c

```diff
@@ -271,6 +272,23 @@ main_init_resources(void)
 #endif /* OC_SOFTWARE_UPDATE */
 }
 
+#ifdef OC_SECURITY
+static void
+main_set_resource_defaults(size_t device)
+{
+  oc_storage_data_clear("u_ids", device);
+  oc_storage_data_clear("pstat", device);
+  oc_storage_data_clear("cred", device);
+  oc_storage_data_clear("acl", device);
```

Comment:
Pls could you also create the macros for the names similar to:
`OCF_SEC_<ID>_STORE_NAME` ?

Reply:
Absolutely.